tests/functional/test_matrix_service.py

```diff
@@ -179,72 +255,43 @@ def _post_notification(self, bundle_fqid, event_type):
-        self._compare_metrics(direct_metrics, mtx_metrics)
-
-    def _analyze_csv_matrix_results(self, request_id, input_bundles):
-        direct_metrics = validation.calculate_ss2_metrics_direct(input_bundles)
-
-        matrix_location = self._retrieve_matrix_location(request_id)
-        self.assertEqual(matrix_location.endswith("csv.zip"), True)
-        csv_metrics = validation.calculate_ss2_metrics_csv(matrix_location)
-        self._compare_metrics(direct_metrics, csv_metrics)
+class TestMatrixServiceV1(MatrixServiceTest):
+    def setUp(self):
+        self.dss_env = MATRIX_ENV_TO_DSS_ENV[os.environ['DEPLOYMENT_STAGE']]
+        self.api_url = f"https://{os.environ['API_HOST']}/v1"
+        self.headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+        self.verbose = True
+        self.s3_file_system = s3fs.S3FileSystem(anon=False)
 
-    def _compare_metrics(self, metrics_1, metrics_2):
-        for metric in metrics_1:
-            delta = metrics_1[metric] / 100000
-            self.assertAlmostEqual(
-                metrics_1[metric], metrics_2[metric], delta=delta,
-                msg=(f"Metric {metric} doesn't match: {metrics_1[metric]} "
-                     f"{metrics_2[metric]}"))
+    def _post_matrix_service_request(self, filter_, fields=None, feature=None, format_=None):
+        data = {"filter": filter_}
+        if fields:
+            data["fields"] = fields
+        if feature:
+            data["feature"] = feature
+        if format_:
+            data["format"] = format_
 
-    def _retrieve_matrix_location(self, request_id):
-        url = f"{self.api_url}/matrix/{request_id}"
-        response = self._make_request(description="GET REQUEST TO MATRIX SERVICE WITH REQUEST ID",
-                                      verb='GET',
-                                      url=url,
-                                      expected_status=200,
+        response = self._make_request(description="POST REQUEST TO MATRIX SERVICE",
+                                      verb='POST',
+                                      url=f"{self.api_url}/matrix",
+                                      expected_status=202,
+                                      data=json.dumps(data),
                                       headers=self.headers)
-        data = json.loads(response)
-        location = data["matrix_location"]
-        return location
 
-    def _make_request(self, description, verb, url, expected_status=None, **options):
-        print(description + ": ")
-        print(f"{verb.upper()} {url}")
-
-        method = getattr(requests, verb.lower())
-        response = method(url, **options)
+        data = json.loads(response)
+        return data["request_id"]
 
-        print(f"-> {response.status_code}")
-        if expected_status:
-            self.assertEqual(expected_status, response.status_code)
+    def test_mtx_output_matrix_service(self):
```

Comment:
To be clear, all previous functional tests are preserved on v0 and v1 is tested with 1 mtx request?